lld/MachO/SyntheticSections.cpp

```diff
@@ -894,8 +940,22 @@ void ObjCStubsSection::writeTo(uint8_t *buf) const {
   uint64_t stubOffset = 0;
   for (size_t i = 0, n = symbols.size(); i < n; ++i) {
     Defined *sym = symbols[i];
+    uint64_t selBaseAddr;
+    uint64_t selIndex;
+
+    auto methname = getMethName(sym);
```

Comment:
nit: we should standardize on capitalization... if it's `selref` and not `selRef`, then this should be `methname` not `methName`

Reply:
I changed it to getMethname in the previous PR https://github.com/llvm/llvm-project/pull/79726.